src/pages/home-page/home-page.css

```diff
@@ -17,19 +17,29 @@
   text-decoration: none;
   color: black;
 }
+.left-nav{
+  width: 50%;
+}
+.category-flex{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+  row-gap: 70px;
```

Comment:
can use rem instead of px for higher values.

Reply:
at some plces px is used intentionally to maintain the UI  for smaller screen